packages/docmanager/src/manager.ts

```diff
@@ -592,6 +592,7 @@ export class DocumentManager implements IDocumentManager {
 
     // If the initial opening of the context fails, dispose of the widget.
     ready.catch(err => {
+      console.error(err)
```

Comment:
Adding some context to the error would be nice

```suggestion
      console.error(`Failed to initialize the context with '${factory.name}' for ${path}`, err)
```